### ISSUE TITLE
Bad URL on get_audio_url(url)

### ISSUE THREAD

**Issue by quesman1** [KartikTalwar/Duolingo#29]
I'm using this project to download word lists for users; some of this data involves downloading the audio using get_audio_url(url). For most words, the URL given works; for others, I get an error'd XML page. Some of these errors are simply due to capitalization -- for example, 'Junge' doesn't work, but 'junge' does. 'Mann' doesn't work, but 'mann' does. Etc. Most of these, I can account for by using a simple try-catch which tests the lowercase version if the regular url returned by get_audio_url() fails. However, there are other URLs that don't work at all.

https://d7mj4aqfscim2.cloudfront.net/tts/de/token/kleide
https://d7mj4aqfscim2.cloudfront.net/tts/de/token/kleidungen
https://d7mj4aqfscim2.cloudfront.net/tts/de/token/guten%20morgen
https://d7mj4aqfscim2.cloudfront.net/tts/de/token/fresse

Neither by making them upper, lower, a mix, it doesn't work. I honestly don't know how these URLs are arrived at in the first place, so there wasn't much troubleshooting I could think of. I know Duolingo has audio of these because I have heard them, so it can't be that they don't exist.

Any ideas?

**Comment by KartikTalwar** [comment]
Can you open chrome's devtools and click on the network tab. When you press the button to hear the audio for one of those words, can you see what URL gets requested. We can debug from there accordingly

**Comment by quesman1** [comment]
Interesting -- when I try clicking on the button for "guten morgen," I get a red link with a 403 Forbidden error. The URL given is https://d7mj4aqfscim2.cloudfront.net/tts/de/token/guten%20morgen. This is a bad XML page. Also of note, the other three words/links I posted (kleide, kleidungen, fresse) don't exist on Duolingo's "Words" page at all.

I decided to go through my code and made note of all failed words. Here's how they break down, with some additions I noticed along the way:

Non-existent on Duolingo.com/words: fresse, baume, kleidungen, hüten, kleide, monds, meere, hemdes

Exists on Duolingo.com/words, but bad link: wie geht's, guten morgen, guten tag

Exists on Duolingo, not detected by get_known_words("de"): wie geht's gut, es tut mir leidt, auf widersehen --> basically, seems like anything that is considered by Duolingo to be a phrase, is not downloaded by get_known_words. I didn't see a method around this that would retrieve phrases; these phrases are, however, listed in the list of vocabulary on Duolingo.com/words.
